changelog.d/20250818_171014_simplify_custom_requests_in_sdk.md

```diff
@@ -0,0 +1,4 @@
+### Changed
+
+- \[SDK\] simplified sending of custom requests with `ApiClient`
```

Comment:
Could you clarify how it was simplified? I don't quite understand the difference.

Reply:
Added documentation on how to do this, added function parameter descriptions and type annotations, `ApiClient.update_params_for_auth()` got several default values for parameters. Actually, changes in `update_params_for_auth()` are the most important part here, because prior to the path you had to call it like [this](https://github.com/humanprotocol/human-protocol/blob/21209ccfadc51db1a9e9d71c5dad8f0e94554094/packages/examples/cvat/exchange-oracle/src/cvat/api_calls.py#L124-L140), which is hard to be called convenient.

Reply:
> Actually, changes in update_params_for_auth() are the most important part here

Okay, then would you mind describing that in the changelog?

Reply:
Changelog is not a good place for the full change list. I've updated the changelog message and the PR description.